
Running tests, synchronously...

Bar:
[Execution Time: 2.97ms]
	Free Test
	[Execution Time: 2.97ms]
		Standard Top Level Checks:                 2;  Failures:  0  [Deep checks: 1]

******  Critical Failures:  1  ******


	Error -- Unexpected Exception:
	Throw after check

	Exception thrown after check completed
	Last Recorded Message:

	Phoney equality check
	[int]

	=======================================

Foo:
[Execution Time: 2.87ms]
	Unit Test
	[Execution Time: 2.87ms]
		Standard Top Level Checks:                 2;  Failures:  0  [Deep checks: 0]

******  Critical Failures:  1  ******


	Error -- Unexpected Exception:
	This is bad

	Exception thrown during last check
	Last Recorded Message:

	Throw during check
	[sequoia::testing::(anonymous namespace)::foo]

	=======================================


-----------Grand Totals-----------
[Total Run Time: 13.4ms]
[Execution Time: 5.84ms]
	Standard Top Level Checks:                 4;  Failures:  0  [Deep checks: 1]
	Standard Performance Checks:               0;  Failures:  0
	False Negative Checks:                     0;  Failures:  0
	False Negative Performance Checks:         0;  Failures:  0
	False Positive Checks:                     0;  Failures:  0
	False Positive Performance Checks:         0;  Failures:  0

******  Critical Failures:  2  ******

